python/cugraph-pyg/pyproject.toml

```diff
@@ -31,30 +31,29 @@ classifiers = [
     "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
+    "cupy-cuda12x>=13.2.0",
     "numpy>=1.23,<3.0a0",
+    "packaging",
     "pandas",
     "pylibcugraph==25.10.*,>=0.0.0a0",
     "pylibwholegraph==25.10.*,>=0.0.0a0",
     "torch-geometric>=2.5,<2.7",
 ] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
 
 [project.urls]
-Homepage = "https://github.com/rapidsai/cugraph"
+Homepage = "https://github.com/rapidsai/cugraph-gnn"
 Documentation = "https://docs.rapids.ai/api/cugraph/stable/"
```

Comment:
Just a side note, we should probably update this. The title tag on this page right now is "`* NOTICE *`", for the DGL deprecation. That is currently very loud, both visually and semantically.